app/src/main/res/values/dimens.xml

```diff
@@ -6,4 +6,41 @@
     <dimen name="nav_header_height">176dp</dimen>
     <dimen name="bottom_sheet_corner_radius">16dp</dimen>
     <dimen name="circle_image_border">2dp</dimen>
-</resources>
\ No newline at end of file
+
+    <!-- margin specifics -->
+    <dimen name="size_point5dp">0.5dp</dimen>
```

Comment:
Only Add which are currently required and naming should be done like text_size_medium,text_size_large,padding_small